packages/auth0-functions/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@asap-hub/auth0-functions",
+  "private": true,
+  "version": "0.1.0",
+  "engines": {
+    "node": "12.x"
+  },
+  "main": "build/index.js",
```

Comment:
These do not exist - we can remove the fields or (this is my preference) reduce te app to just this function in an `index.ts`, and when we create another function, just create a different app for it